crates/control/src/behavior/initial.rs

```diff
@@ -34,10 +36,13 @@ fn look_at_referee(
     ground_to_field: Option<Isometry2<Ground, Field>>,
     filtered_game_controller_state: Option<FilteredGameControllerState>,
     expected_referee_position: Option<Point2<Field>>,
+    enable_pose_detection: bool,
 ) -> Option<MotionCommand> {
     let ground_to_field = ground_to_field?;
     let expected_referee_position = expected_referee_position?;
-    if filtered_game_controller_state?.game_state != FilteredGameState::Initial {
+    if !enable_pose_detection
+        && filtered_game_controller_state?.game_state != FilteredGameState::Initial
```

Comment:
Shouldn't this be 
```suggestion
        || filtered_game_controller_state?.game_state != FilteredGameState::Initial
```
???

Reply:
You are absolutely right. Done, but this change as well as the move of the pose detection toggle is now tracked in #1093.